mobile/src/main/java/rs/readahead/washington/mobile/domain/entity/reports/TellaReportServer.kt

```diff
@@ -0,0 +1,43 @@
+package rs.readahead.washington.mobile.domain.entity.reports
+
+import rs.readahead.washington.mobile.domain.entity.Server
+import rs.readahead.washington.mobile.domain.entity.ServerType
+import java.io.Serializable
+
+class TellaReportServer @JvmOverloads constructor(
+    id: Long = 0,
+    var accessToken: String = "",
+    var projectId: String = "",
+    var projectName: String = "",
+    var projectSlug: String = "",
+    var isActivatedMetadata: Boolean = false,
+    var isActivatedBackgroundUpload: Boolean = false
+) :
+    Server(), Serializable {
+    override fun equals(other: Any?): Boolean {
+        if (other == null) {
+            return false
+        }
+        if (this === other) {
+            return true
+        }
+        if (other !is TellaReportServer) {
+            return false
+        }
+        return id == other.id
```

Comment:
This way it is possible to have the same project on the same server as a different connection ("server"). So I guess we should compare project too.